updatecli/weekly.d/contributing.yaml

```diff
@@ -0,0 +1,82 @@
+name: Update Technical Requirements in CONTRIBUTING.md
+
+sources:
+  getMavenLabel:
+    kind: file
+    name: Get Maven label from Jenkinsfile
+    spec:
+      file: Jenkinsfile
+      marker: 'label '''
+      pattern: '(maven-[0-9]+)'
+
+  getPackerImageVersion:
+    kind: yaml
+    name: Get Packer image version from jenkins-infra
+    spec:
+      file: hieradata/common.yaml
+      key: $.profile::jenkinscontroller::jcasc.agent_images.azure_vms_gallery_image.version
+
+  getToolVersions:
+    kind: yaml
+    name: Get tool versions from packer-images
+    dependson:
+      - getPackerImageVersion
+    spec:
+      file: https://raw.githubusercontent.com/jenkins-infra/packer-images/{{ source "getPackerImageVersion" }}/provisioning/tools-versions.yml
+
+conditions:
+  checkMavenAvailability:
+    kind: shell
+    disablesourceinput: true
+    spec:
+      command: curl --connect-timeout 5 --location --head --fail --silent --show-error https://archive.apache.org/dist/maven/maven-3/{{ source "getToolVersions" "maven.version" }}/binaries/apache-maven-{{ source "getToolVersions" "maven.version" }}-bin.tar.gz
+
+  checkJDKAvailability:
+    kind: shell
+    disablesourceinput: true
+    spec:
+      command: curl --connect-timeout 5 --location --head --fail --silent --show-error https://adoptium.net/temurin/releases/v{{ source "getToolVersions" "jdk.version" }}
+
+  checkPackerImageExists:
+    kind: http
+    name: Verify packer-images tools-versions.yml exists
+    spec:
+      url: https://raw.githubusercontent.com/jenkins-infra/packer-images/{{ source "getPackerImageVersion" }}/provisioning/tools-versions.yml
```

Comment:
```suggestion
```

Conditions are not worth the effort in this particular case. In particular:
- The versions tracked by the sources are already maintained through updatecli with the same upstreams conditions
- If a PR is opened on this documentation, it is not a blocker if anything is temporarily unavailable. It's not blocking anything.